app/src/org/commcare/preferences/CommCareServerPreferences.java

```diff
@@ -96,6 +96,10 @@ public static String getSupportEmailAddress() {
         return getServerProperty(PREFS_SUPPORT_ADDRESS_KEY, CommCareApplication.instance().getString(R.string.support_email_address_default)) ;
     }
 
+    public static String getFormRecordKey() {
```

Comment:
Ah yea that's a fair point, but I think you're right that these should both just live and be made visible through Developer Preferences only. While the remote payload URL is technically a server pref in that it represents a URL for server communications, I think it's more important that it be hidden from regular users.